Configs/config.json

```diff
@@ -1,8 +1,11 @@
-{  
-  "settings":{  
-    "AutomaticPluginReloading":true,
-    "AutomaticPluginReloadSeconds":5,
+{
+  "settings": {
+    "AutomaticPluginReloading": true,
+    "AutomaticPluginReloadSeconds": 5,
     "SaveWorldBeforePluginReload": true,
-    "AttachToParent": true
+    "AttachToParent": true,
+    "MaxFiles": 20,
+    "MaxAgeInSeconds": 86400,
+    "EnableDeleteLogging": true
```

Comment:
I don't think we need this.

We can use an info/audit log instead and the end-user can set their log level appropriately.

Reply:
Every user has their preferences how they want to keep their logs, for me while developing a plugin this logs can be large in count as a turning on and off the server more often in a day i can have like more than 200 files though its not that big but imaging a week of server have their daily restart or some sort, if you are referring to the log `EnableDeleteLogging` well that can be remove

Reply:
Yes, I was referring to `EnableDeleteLogging`.